.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on: [pull_request, push]
```

Comment:
Should I make this:
- push to `master` and `release/*`
- pull request

Reply:
Yes, that makes sense! :)